### DISCUSSION TITLE
Download effects from https://patchstorage.com/platform/lv2-plugins/

### DISCUSSION THREAD

**Discussion by zetoni** [rerdavies/pipedal#135] [Ideas]
Today I installed PiPedal and it worked very well for the initial testing. Congratulations for the fantastic work!

It would be great to leverage the LV2 plugins already loaded in patchstorage. Can you plan to integrate PiPedal to it?

Thanks,

Jose Antonio

**Comment by Sedaxel** [comment]
I´ve just installed NAM from patchstorage. Yes, I know there is a Toob Neural Amp Modeler already included in PiPedal, but I noticed the gain  input control not working (at least in my Pi), and not liking the gate. I don´t need the tonestacks included either...

The patchstorage NAM only has gain input, gain output and a button stating "buffered" which I don´t know what it actually does. I like better this simpler layout of NAM.

To install it, I just copied the lv2 folder (extracted from the gz and tar files) into the folder where the system has the plugins (if I remember... I belive it´s usr/bin/lv2... but I´m not sure, just search for it). And that was all, it worked right away.
